v3/integration/config.json

```diff
@@ -599,9 +599,7 @@
     },
     "e_subject_email_max_length": {},
     "e_subject_empty_without_san": {},
-    "e_subject_given_name_max_length": {
-      "ErrCount": 2
-    },
+    "e_subject_given_name_max_length": {},
```

Comment:
Just double checking: I'm assuming the certs here were > 16 and < 64 characters, hence no added warning result?

Reply:
Precisely, the names in question were `Johannes Elias Markus` and `SALVADOR CRISTOBAL`.